test/contracts/promises/child-simple/wasm/Cargo.lock

```diff
@@ -0,0 +1,185 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Cargo.lock is normally added to `.gitignore`

Reply:
Removed multiple `Cargo.lock` files and added to `.gitignore`.